install/empty_data.php

```diff
@@ -0,0 +1,7994 @@
+      'use_reservations_alert'               => 0,
+      'autoclose_delay'                      => -10,
+      'notclosed_delay'                      => 0,
+      'calendars_id'                         => 0,
+      'auto_assign_mode'                     => -10,
+      'tickettype'                           => 1,
+      'inquest_config'                       => 1,
+      'inquest_rate'                         => 0,
+      'inquest_delay'                        => 0,
+      'autofill_warranty_date'               => 0,
+      'autofill_use_date'                    => 0,
+      'autofill_buy_date'                    => 0,
+      'autofill_delivery_date'               => 0,
+      'autofill_order_date'                  => 0,
+      'tickettemplates_id'                   => 1,
+      'entities_id_software'                 => -10,
+      'default_contract_alert'               => 0,
+      'default_infocom_alert'                => 0,
+      'default_cartridges_alarm_threshold'   => 10,
+      'default_consumables_alarm_threshold'  => 10,
+      'delay_send_emails'                    => 0,
+      'is_notif_enable_default'              => 1,
+      'autofill_decommission_date'           => 0
+   ]
+];
+
+$tables['glpi_filesystems'] = [
+    ['id' => 1, 'name'  => 'ext'],
+    ['id' => 2, 'name'  => 'ext2'],
+    ['id' => 3, 'name'  => 'ext3'],
+    ['id' => 4, 'name'  => 'ext4'],
+    ['id' => 5, 'name'  => 'FAT'],
+    ['id' => 6, 'name'  => 'FAT32'],
+    ['id' => 7, 'name'  => 'VFAT'],
+    ['id' => 8, 'name'  => 'HFS'],
+    ['id' => 9, 'name'  => 'HPFS'],
+    ['id' => 10, 'name'  => 'HTFS'],
+    ['id' => 11, 'name'  => 'JFS'],
+    ['id' => 12, 'name'  => 'JFS2'],
+    ['id' => 13, 'name'  => 'NFS'],
+    ['id' => 14, 'name'  => 'NTFS'],
+    ['id' => 15, 'name'  => 'ReiserFS'],
+    ['id' => 16, 'name'  => 'SMBFS'],
+    ['id' => 17, 'name'  => 'UDF'],
+    ['id' => 18, 'name'  => 'UFS'],
+    ['id' => 19, 'name'  => 'XFS'],
+    ['id' => 20, 'name'  => 'ZFS']
+];
+
+$tables['glpi_interfacetypes'] = [
+   ['id' => 1, 'name' => 'IDE'],
+   ['id' => 2, 'name' => 'SATA'],
+   ['id' => 3, 'name' => 'SCSI'],
+   ['id' => 4, 'name' => 'USB'],
+   ['id' => 5, 'name' => 'AGP'],
+   ['id' => 6, 'name' => 'PCI'],
+   ['id' => 7, 'name' => 'PCIe'],
+   ['id' => 8, 'name' => 'PCI-X']
+];
+
+$tables['glpi_notifications'] = [
+    [
+      'id'           => 1,
+      'name'         => 'Alert Tickets not closed',
+      'itemtype'     => 'Ticket',
+      'event'        => 'alertnotclosed',
+      'is_recursive' => 1,
+      'is_active'    => 1
+    ], [
+      'id'           => 2,
+      'name'         => 'New Ticket',
+      'itemtype'     => 'Ticket',
+      'event'        => 'new',
+      'is_recursive' => 1,
+      'is_active'    => 1
+    ], [
+      'id'           => 3,
+      'name'         => 'Update Ticket',
+      'itemtype'     => 'Ticket',
+      'event'        => 'update',
+      'is_recursive' => 1,
+      'is_active'
```

Comment:
One idea as we are now in a php script.
Could we set some constants to identify 
- profiles id (ex PROF_SUPERADMIN_ID, PROF_OBSERVER_ID, etc)
- users id (USER_GLPI_ID, USER_TECH_ID, etc)
- Rights values (READ, UPDATE, etc)

Reply:
I agree; but this is out of the scope of the current PR, I'd prefer to work on that on a separate one